ifrs17/Import/4ImportScope-TechnicalMargin.ipynb

```diff
@@ -581,9 +631,14 @@
         {
             "cell_type": "markdown",
             "source": [
-                "# Technical Margin allocation for Reinsurance",
+                "# Loss Recovery Component",
                 "\n",
-                "\nIn the case of Reinsurance a Loss Recovery Component is computed when the underlying gross business opens a Loss component. Loss Recovery Component uses the notion of Loss Recovery Component Boundary ([LoReCo Boundary](#loreco-boundary)). "
+                "\nIn the case of Reinsurance a Loss Recovery Component is computed when the underlying gross business opens a Loss component. ",
```

Comment:
Add this: In the case of Reinsurance a Loss Recovery Component is computed when the underlying gross **_LRC_** business opens a Loss component. _If the underlying gross business is LIC, there is no Loss Recovery Component, as there is also no Loss Component in the gross side._